### PR TITLE
Added implementation of network method for class RtDocker and test for it

### PR DESCRIPTION
**Pull request by lumimul**
PR for #167 
Implemented RtDocker.networks()

### CODE REVIEW

File: src/test/java/com/amihaiemil/docker/TcpDockerTestCase.java [hunk 1]
```diff
@@ -171,4 +171,18 @@ public void returnsVolumes() {
...
+     * TcpDocker can return Networks.
+     */
+    @Test
+    public void returnsNetworks() {
+        MatcherAssert.assertThat(
+                new TcpDocker(
+                        Mockito.mock(HttpClient.class),
+                        URI.create("http://localhost")
+                ).networks(),
+                Matchers.notNullValue()
```
Comment (amihaiemil) [inline src/test/java/com/amihaiemil/docker/TcpDockerTestCase.java:185]: @lumimul Can you please indent the lines 4 spaces to the left?

File: src/main/java/com/amihaiemil/docker/RtDocker.java [hunk 2]
```diff
@@ -105,9 +105,10 @@ public final Images images() {
 
     @Override
     public final Networks networks() {
-        throw new UnsupportedOperationException(
-            "Networks API is not yet implemented. If you can contribute please,"
-            + " do it here: https://www.github.com/amihaiemil/docker-java-api"
+        return new ListedNetworks(
+                this.client,
+                URI.create(this.baseUri.toString() + "/networks"),
+                this
```
Comment (amihaiemil) [inline src/main/java/com/amihaiemil/docker/RtDocker.java:111]: @lumimul Same here, 4 spaces to the left please :)

Comment (lumimul) [reply src/main/java/com/amihaiemil/docker/RtDocker.java]: done

File: src/test/java/com/amihaiemil/docker/TcpDockerTestCase.java (hunk 1 above)
Comment (lumimul) [reply src/test/java/com/amihaiemil/docker/TcpDockerTestCase.java:185]: done

**Comment by amihaiemil** [comment]
@rultor merge it please

File: src/test/java/com/amihaiemil/docker/TcpDockerTestCase.java (hunk 1 above)
Comment (lumimul) [reply src/test/java/com/amihaiemil/docker/TcpDockerTestCase.java]: done

**Comment by amihaiemil** [comment]
@lumimul there are some test failures, can you please fix them? Make sure the build ``mvn clean install -Pcheckstyle`` passes :D

**Comment by lumimul** [comment]
> @lumimul there are some test failures, can you please fix them? Make sure the build `mvn clean install -Pcheckstyle` passes :D

yes, sorry, fix it

**Comment by amihaiemil** [comment]
@rultor merge it please

**Comment by amihaiemil** [comment]
@rultor try to merge again

**Comment by 0crat** [comment]
Job `gh:amihaiemil/docker-java-api#330` is not assigned, can't get performer

**Comment by amihaiemil** [comment]
@lumimul Thanks for your contribution!

**Pull request merged**